cmd/sso.go

```diff
@@ -0,0 +1,128 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	ssoCmd = &cobra.Command{
+		GroupID: groupManagementAPI,
+		Use:     "sso",
+		Short:   "Manage Single Sign-On (SSO) authentication for projects",
+	}
+
+	ssoProjectRef           string
+	ssoMetadataFile         string
+	ssoMetadataURL          string
+	ssoAttributeMappingFile string
+	ssoDomains              []string
+	ssoAddDomains           []string
+	ssoRemoveDomains        []string
+
+	ssoAddCmd = &cobra.Command{
+		Use:     "add <type> [flags]",
+		Short:   "Add a new SSO identity provider",
+		Args:    cobra.ExactArgs(1),
+		Example: `  supabase sso add saml --project-ref abcdefghijklmn --metadata-file ~/SAMLMetadata.xml`,
```

Comment:
Would <type> here refer to the SSO method? (e.g. SAML, OIDC, password SSO etc) ?

Reply:
Sorry what's "here" 🙃

Reply:
Oops that should be `type`, but looks like it's been added with a default of saml.

Reply:
i dont think this defaults to saml - if it's excluded, i get the following error: `Error: accepts 1 arg(s), received 0`